config.json

```diff
@@ -660,6 +660,16 @@
       ]
     },
     {
+      "uuid": "c21c379b-fb23-449b-809a-3c6ef1c31221",
+      "slug": "pig-latin",
```

Comment:
Since the exercises will be delivered in the order the appear in this `exercises` array, it seems to make the most sense to move this exercise somewhere with the other difficulty 4 exercises.